test/evergreen.yml

```diff
@@ -555,8 +558,9 @@ functions:
               if [ -d cmake_build ]; then rm -r cmake_build; fi
               mkdir -p cmake_build
               pushd cmake_build
-              # Adding -DENABLE_PYTHON=1 -DENABLE_STRICT=1 as 6.0 does not default these like develop.
-              $CMAKE -DCMAKE_C_FLAGS="-DMIGHT_NOT_RUN -Wno-error" -DENABLE_PYTHON=1 -DENABLE_STRICT=1 ../.
+              # Adding -DENABLE_PYTHON=ON -DENABLE_STRICT=ON as 6.0 does not default these like develop.
+              # Disable PALite build, as docs build does not need it and it can fail to build on some platforms.
+              $CMAKE -DCMAKE_C_FLAGS="-DMIGHT_NOT_RUN -Wno-error" -DENABLE_PYTHON=ON -DENABLE_STRICT=ON -DENABLE_PALITE=OFF ../.
```

Comment:
ditto

Reply:
This implies default is DENABLE_PALITE=ON, instead i only want to turn on PALITE=ON when we need to use disagg.

Reply:
Right now we cannot distinguish between disagg and non-disagg tests/builds. The `unit-test*` tasks run everything.
It's much bigger project to separate disagg and non-disagg builds. The practical solution is to disable PALite for certain builds where no tests expected to run.